index.js

```diff
@@ -208,8 +234,11 @@ FocusGroup.prototype.addMember = function(memberData, index) {
 
   var cleanedNodeText = nodeText.replace(/[\W_]/g, '').toLowerCase();
   var member = {
+    id: memberData.id,
+    index: index,
     node: node,
     text: cleanedNodeText,
+    value: memberData.value
```

Comment:
Instead of adding the specific properties `id` and `value`, how about adding a single property `data` that can accept any arbitrary value — e.g. an object with `id` and `value`?